charts/kargo/resources/crds/kargo.akuity.io_stages.yaml

```diff
@@ -286,15 +286,27 @@ spec:
                         a required field.
                       properties:
                         availabilityStrategy:
+                          allOf:
+                          - enum:
+                            - All
+                            - OneOf
+                            - ""
+                          - enum:
+                            - All
+                            - OneOf
+                            - ""
```

Comment:
I notice this is duplicated now. I think I know why... I'll link to here from the spot where I think it happened.